components/Typography/src/heading.scss

```diff
@@ -22,10 +22,54 @@
   font-family: var(--utrecht-heading-font-family);
 }
 
+.utrecht-heading-1:first-child {
+  /* overwriting root :root .utrecht-heading-1:first-child */
+  --utrecht-heading-1-margin-block-start: var(--denhaag-space-block-xs, 0.5rem) !important;
+}
+
+* + .utrecht-heading-2 {
```

Comment:
`*` is every component including the columns which already have a spacing. I would suggest here a list of `component:not(.component) +.utrecht-heading-X`.

Components like a button-group,  accordion-group, cta-link-group already have spacings around them, i think this will cause an issue.

Did you check how this would look in the WP environment on a page with all possible combinations?

Reply:
@paulAcato isn't that a wordpress theme specific exception?
I can put that in the theme itself like this: `*:not(.denhaag-columns):not(.denhaag-button-group) + .utrecht-heading-2`

Reply:
@robbieatacato , in my opinion not. Since the columns, accordion and cta-link-group are all in the nl-design-system of  the Hague. If u apply it with *, i wish you luck with overwriting all these styles.

Reply:
@paulAcato I still cannot find these css classes with margins: 

- .accordion-group / .denhaag-accordion-group
- .cta-link-group / .denhaag-cta-link-group

If you mean:

- .denhaag-accordion, this component doesn't have spacing at the bottom (https://nl-design-system.github.io/denhaag/?path=/story/css-surfaces-accordion--default-story)
- .denhaag-cta-link, this component doesn't have spacing at the bottom (https://nl-design-system.github.io/denhaag/?path=/story/css-actions-cta-link--grouped)